bin/branding.list

```diff
@@ -10,6 +10,7 @@ AUTHOR
 LICENSE
 HOMEPAGE
 CHANGELOG_PAGE
+ORGANIZATION
```

Comment:
This  should be added to the legal empty variables in python branding. 
Unless you're publishing, it's not required